ereuse_devicehub/resources/action/views/snapshot.py

```diff
@@ -74,8 +78,12 @@ def __init__(self, snapshot_json: dict, resource_def, schema):
         self.tmp_snapshots = app.config['TMP_SNAPSHOTS']
         self.path_snapshot = save_json(snapshot_json, self.tmp_snapshots, g.user.email)
         snapshot_json.pop('debug', None)
-        self.snapshot_json = resource_def.schema.load(snapshot_json)
-        self.response = self.build()
+        if snapshot_json.get('version') in ["2022.03"]:
```

Comment:
See previous comment about hardcofing constants.

Reply:
is possible refactor a little too. Done